### PR TITLE
CONSOLE-3124: migrate Cluster Dashboard test to Cypress

### PR DESCRIPTION
**Pull request by rhamilto**
We are continuing to see high flake rates with the Protractor version of the Cluster Dashboard test, so migrating it to Cypress should help abate the flakes.  Additionally, this re-enables the Details Card test, which was previously disabled.

### CODE REVIEW

**Comment by rhamilto** [comment]
This story is tech debt and has no user-facing impact:

/px-approved
/docs-approved

**Comment by jhadvig** [comment]
Addint he `qe-approved` label manually to see the actual outcome of the test's flakiness on other console's PR, instead of retesting this one.

**Comment by rhamilto** [comment]
Removed `.click()`s to avoid flakes resulting from https://github.com/cypress-io/cypress/issues/7306.

@jhadvig, PTAL

File: frontend/packages/integration-tests-cypress/tests/dashboards/cluster-dashboard.spec.ts [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+          .eq(i)
+          .should('have.text', title);
+      });
+      const expectedValues = [
+        { assertion: 'include.text', value: 'https://' },
+        { assertion: 'include.text', value: '-' },
+        { assertion: 'not.to.be.empty' },
+        { assertion: 'include.text', value: '.' },
+        { assertion: 'not.to.be.empty' },
+        { assertion: 'include.text', value: '-' },
```
Comment (rhamilto) [inline frontend/packages/integration-tests-cypress/tests/dashboards/cluster-dashboard.spec.ts:39]: Oh, yeah.  Clusters in CI don't have a channel configured.  Will fix.

**Comment by rhamilto** [comment]
@jhadvig, take 3.  I tested against a fresh cluster-bot cluster this time.  I think we're good.  🤞 Please re-tag.

**Pull request merged**